ios/browser/api/favicon/favicon_driver.h

```diff
@@ -10,16 +10,21 @@
 #import <UIKit/UIKit.h>
 #import <Webkit/Webkit.h>
 
+@class WebState;
+
 NS_ASSUME_NONNULL_BEGIN
 
 OBJC_EXPORT
 NS_SWIFT_NAME(FaviconLoader.Driver)
-@interface BraveFaviconDriver : NSObject
-- (instancetype)initWithPrivateBrowsingMode:(bool)privateMode;
+@interface FaviconDriver : NSObject
```

Comment:
Are the swift name and obj-c supposed to be mismatched on purpose?

Reply:
Yes. Intentional. I wanted to export the driver as part of the loader instead of two separate independent exports.
So the usage should be: `FaviconLoader` but each loader contains a `driver` within it. I guess I could change it to be separate exports but not sure if I should.